pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/ElasticSearchClient.java

```diff
@@ -56,8 +58,18 @@ public class ElasticSearchClient implements AutoCloseable {
     final AtomicReference<Exception> irrecoverableError = new AtomicReference<>();
     private final IndexNameFormatter indexNameFormatter;
 
-    public ElasticSearchClient(ElasticSearchConfig elasticSearchConfig) {
+    // sink metrics
+    public static final String METRICS_TOTAL_INCOMING = "_elasticsearch_total_incoming_";
```

Comment:
1. The metric name goes as a label into a Prometheus Summary collector. Why the `_` prefix?
2. I would remove `_total_` everywhere. This is just a label in a Prometheus Summary, so this ends up exported as multiple quantiles and sum and count. So maybe let's just express what this counts.

Reply:
I follow the same pattern from KinesisSink.java. I suppose the there is a prefix to be added to this metrics name. The leading underscore is a delimiter.
I can remove the _total

Reply:
You can double-check me - run the connector and check the `/metrics` output. If I'm right, all I'm saying is previous code shouldn't be used as an example.
Removing the total is only 2nd point. Look at point 1 as well.

Reply:
> 1. The metric name goes as a label into a Prometheus Summary collector. Why the `_` prefix?

Great point @asafm. I had assumed these were metrics names, not label names. Here is the generation code:

https://github.com/apache/pulsar/blob/b1822ed2b4714e4b485ab695eb1d52634ecc8db4/pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/ContextImpl.java#L212-L222

We should remove the prefix and suffix of the `_`.

Reply:
Removed both leading and ending underscores